jekyll/_cci2/faq.md

```diff
@@ -317,14 +317,12 @@ To find a list of your Active Users, log in to the CircleCI web app > click `Pla
 #### What happens when I run out of credits?
 {:.no_toc}
 
-On the Performance plan, when you reach 10% of your remaining credits, you will
-be refilled 25% of your credit subscription (at 25% the cost of your plan). For
-example, If your monthly package size is 25,000 credits, you will automatically
-be refilled 6,250 credits (at $.0006 each, not including applicable taxes) when
-you reach 2,500 remaining credits.
+On the **Performance plan**, when you reach 2% of your remaining credits, you will be refilled 25% of your credit subscription (at 25% the cost of your plan). For example, If your monthly package size is 25,000 credits, you will automatically be refilled 6,250 credits (at $.0006 each, not including applicable taxes) when you reach 500 remaining credits.
```

Comment:
@rosieyohannan can you update to:
On the **Performance plan**, when you reach 2% of your remaining credits, you will be refilled 25% of your credit subscription, with a minimum refill of 25,000 credits. For example, If your monthly package size is 100,000 credits, you will automatically be refilled 25,000 credits (at $.0006 each, not including applicable taxes) when you reach 2000 remaining credits.